server/utils/assessmentUtils.test.ts

```diff
@@ -0,0 +1,94 @@
+import { AssessmentWithRisks } from '@approved-premises/ui'
+import { daysSinceReceived, getStatus, formattedArrivalDate, awaitingAssessmentTableRows } from './assessmentUtils'
+import { DateFormats } from './dateUtils'
+
+import * as personUtils from './personUtils'
+import * as applicationUtils from './applicationUtils'
+
+import assessmentFactory from '../testutils/factories/assessment'
+import risksFactory from '../testutils/factories/risks'
+import { tierBadge } from './personUtils'
+
+jest.mock('./applicationUtils')
+jest.mock('./personUtils')
+
+describe('assessmentUtils', () => {
+  beforeEach(() => {
+    jest.resetAllMocks()
+  })
+
+  describe('daysSinceReceived', () => {
+    it('returns the difference in days since the assessment has been received', () => {
+      const assessment = assessmentFactory.createdXDaysAgo(10).build()
+
+      expect(daysSinceReceived(assessment)).toEqual(10)
+    })
+  })
+
+  describe('daysUntilDue', () => {
```

Comment:
Looks like you've got the test for this in ahead of the function itself - function itself is in the next commit